build.gradle.kts

```diff
@@ -72,7 +75,7 @@ dependencies {
 
   implementation("io.github.microutils:kotlin-logging:$kotlinLoggingVersion")
 
-  runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("org.flywaydb:flyway-database-postgresql")
```

Comment:
Updated to resolve breaking change,
https://github.com/ministryofjustice/dps-gradle-spring-boot/blob/main/release-notes/6.x.md#breaking-changes